### ISSUE TITLE
The Firefox add-on is out of date

### ISSUE THREAD

**Issue by kbdragon** [ClearURLs/Addon#395]
The add-on page for ClearURLs shows that it was last updated on Jan 31, 2023 - almost two years ago. In this repo I see progress on very much needed features (whitelist). 
is there any way to download an xpi or update the addon?

https://addons.mozilla.org/en-GB/firefox/addon/clearurls/

**Comment by Hermholtz** [comment]
Yes, please update the Firefox addon. Really hoping that recent changes will resolve many open issues.
Thanks in advance!

**Comment by sncix** [comment]
Look like it has been updated now. (mine was auto-updated to 1.27.1 on 16 Jan)

**Comment by Hermholtz** [comment]
The issue can be closed now.

**Comment by KevinRoebert** [comment]
> Today it shows an old version but today's update date, hence I have no idea which version I have now. Therefore the issue is still valid.
> 
> ![Image](https://github.com/user-attachments/assets/4faac635-56a9-4765-98dc-c6eb1993bb0d)

That's right. I have deactivated the broken version 1.27.1. The current version 1.27.2 has already been submitted and is currently under review at Mozilla.
